src/models/transactions/common.ts

```diff
@@ -48,6 +48,17 @@ const isSigner = (signer: Signer): boolean => {
         && typeof signer.SigningPubKey === 'string'
 }
 
+export function isIssuedCurrency (obj: IssuedCurrencyAmount): boolean {
+    return Object.keys(obj).length === 3 
+        && typeof obj.value === 'string'
+        && typeof obj.issuer === 'string'
+        && typeof obj.currency === 'string'
+}
```

Comment:
You should use this function in `validateOfferCreate` as well, which creates its own

Reply:
Done!